src/components/views/settings/SpellCheckSettings.tsx

```diff
@@ -18,10 +18,11 @@ import React from "react";
 
 import SpellCheckLanguagesDropdown from "../../../components/views/elements/SpellCheckLanguagesDropdown";
 import AccessibleButton, { ButtonEvent } from "../../../components/views/elements/AccessibleButton";
-import { _t } from "../../../languageHandler";
+import { _t, getUserLanguage } from "../../../languageHandler";
 
 interface ExistingSpellCheckLanguageIProps {
     language: string;
+    label?: string;
```

Comment:
documentation, please. I know this isn't an exported class, but it still represents the public interface to `ExistingSpellCheckLanguage` and we should be in the habit of documenting all of our props.

Please include the behavior if `label` is omitted in the doc.